internals/secrethub/debug.go

```diff
@@ -8,7 +8,7 @@ import (
 // RegisterDebugFlag registers a debug flag that changes the log level of the given logger to DEBUG.
 func RegisterDebugFlag(app *cli.App, logger cli.Logger) {
 	app.PersistentFlags().BoolP("debug", "D", false, "Enable debug mode.")
-	app.Root.Cmd.PersistentPreRunE = cli.MergeFuncs(app.Root.Cmd.PersistentPreRunE, func(command *cobra.Command, strings []string) error {
+	app.Root.AddPreRunE(func(command *cobra.Command, strings []string) error {
```

Comment:
This was `PersistentPreRunE`, same in `mlock.go`.

Reply:
My bad. Fixed.